CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+## Unreleased - as of Sprint 53 end 2017-01-30
```

Comment:
That's a good point.  When was last release of this?  was there one?  If you can get me that info, I can add in all the things that have happened since last release.